src/utils/api/calls.js

```diff
@@ -149,10 +152,17 @@ exports.getTutors = async function getTutors() {
 };
 
 async function getAccessToken() {
+	const cacheContent = getCache(key);
```

Comment:
Love to see this! Quick note: The Access Token expires after 5 minutes. When does this cache clear/reset? Will we need to add logic to clear cache & re-fetch access token if we receive a 401 error from CougarCS Cloud? (401 error returned if token is expired, OR if auth token is missing in request)

Reply:
Alternatively, do you have recommendations on token expiry duration? I can extend it to match the cache reset timing if that is better design

Reply:
In prod its 4 hrs, I looked at this based on the traffic we get